modules/digitaluniverse/rendering/renderableplanescloud.cpp

```diff
@@ -1307,45 +1249,34 @@ void RenderablePlanesCloud::createPlanes() {
             u *= _scaleFactor;
             v *= _scaleFactor;
 
-            RenderingPlane plane;
-            plane.planeIndex = static_cast<int>(_fullData[p + _textureVariableIndex]);
-
-            // JCC: Ask Abbott about these points refeering to a non-existing texture.
-            if (plane.planeIndex == 30) {
-                plane.planeIndex = -1;
-            }
-
-            glGenVertexArrays(1, &plane.vao);
-            glGenBuffers(1, &plane.vbo);
-
             glm::vec4 vertex0 = transformedPos - u - v; // same as 3
             glm::vec4 vertex1 = transformedPos + u + v; // same as 5
             glm::vec4 vertex2 = transformedPos - u + v;
             glm::vec4 vertex4 = transformedPos + u - v;
 
             float scale = 0.f;
             switch (_unit) {
-                case Meter:
-                    scale = 1.f;
-                    break;
-                case Kilometer:
-                    scale = 1e3f;
-                    break;
-                case Parsec:
-                    scale = static_cast<float>(PARSEC);
-                    break;
-                case Kiloparsec:
-                    scale = static_cast<float>(1e3 * PARSEC);
-                    break;
-                case Megaparsec:
-                    scale = static_cast<float>(1e6 * PARSEC);
-                    break;
-                case Gigaparsec:
-                    scale = static_cast<float>(1e9 * PARSEC);
-                    break;
-                case GigalightYears:
-                    scale = static_cast<float>(306391534.73091 * PARSEC);
-                    break;
+            case Meter:
```

Comment:
The old style was good :)

Reply:
The copy/paste removed the style. Fixed.